src/js/components/sharedComponents/FooterLinkToAdvancedSearch.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * AgencyFooter.jsx -> FooterLinkToAdvancedSearch.jsx
+ * Created by Kevin Li 6/16/17
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    clickedSearch: PropTypes.func,
+    title: PropTypes.string,
+    description: PropTypes.string
+};
+
+const FooterLinkToAdvancedSearch = ({
+    clickedSearch,
+    title,
+    description
+}) => (
+    <div className="footerLinkToAdvancedSearch">
+        <div className="footerLinkToAdvancedSearch-content">
```

Comment:
Comment above about [BEM](http://getbem.com/naming/), may need to update this to `footerLinkToAdvancedSearch__content`.